cypress/commons/actions/brewery/BreweryParameters.js

```diff
@@ -108,18 +108,21 @@ function getAverageConsumptionInput() {
   return getAverageConsumption().find(GENERIC_SELECTORS.genericComponents.basicSliderInput.input);
 }
 function getAverageConsumptionSlider() {
-  return getAverageConsumption().find('[role=slider]');
+  return getAverageConsumption().find(GENERIC_SELECTORS.genericComponents.basicSliderInput.root);
 }
 function moveAverageConsumptionSlider(sliderMovement) {
-  getAverageConsumptionInput().then((input) => {
-    if (parseInt(input[0].value) === sliderMovement) return;
-    const sliderStepsToMove = sliderMovement - parseInt(input[0].value);
-    const arrowsCount =
-      sliderStepsToMove > 0
-        ? '{rightArrow}-'.repeat(Math.abs(sliderStepsToMove))
-        : '{leftArrow}-'.repeat(Math.abs(sliderStepsToMove));
-    getAverageConsumptionSlider().click().type(arrowsCount);
-  });
+  getAverageConsumptionSlider()
+    .invoke('width')
+    .then((value) => {
+      const x = (value * sliderMovement) / 10;
```

Comment:
Nice, cypress never stops surprising me :rofl: Just a question: you are using 0 and 10 because these are min and max values of `sliderValue` ? If it is so, maybe we can add min and max as arguments to the function in order to be able to reuse it elsewhere ?

Reply:
We can.